docusaurus/docs/reactnative/guides/push_notifications_v2.mdx

```diff
@@ -519,7 +541,7 @@ useEffect(() => {
 
 ## Troubleshooting
 
-- You can check the "Webhook & Push Logs" section in the Dashboard to see if Notifications were sent by Stream. Please see the [Common Issues & FAQ](https://getstream.io/chat/docs/javascript/push_-_common_issues_and_faq/?language=javascript) to diagnose the various cases in which the notifications may not be sent by the Stream Chat API.
+- You can check the "Webhook & Push Logs" section in the Dashboard to see if Notifications were sent by Stream. Please see the [Common Issues & FAQ](https://getstream.io/chat/docs/javascript/push_-_common_issues_and_faq/?language=javascript) to diagnose the various cases in which the Stream Chat API may not send the notifications.
 - Android devices can restrict your app while it's in the background to preserve battery life. This can prevent trigger notifications from being displayed. The [Notifee documentation](https://notifee.app/react-native/docs/android/background-restrictions) explains it in detail.
-- Additionally, if Stream Chat API has sent the notifications but was not displayed on the device, you can check the diagnostics steps for [Android](https://firebase.google.com/support/troubleshooter/fcm/delivery/diagnose/android) and [iOS](https://firebase.google.com/support/troubleshooter/fcm/delivery/diagnose/ios) in the Firebase cloud messaging documentation.
-- If you are still having trouble with Push Notifications, please contact [support@getstream.io](support@getstream.io).
+- Additionally, if Stream Chat API has sent the notificationsˀ but was not displayed on the device, you can check the diagnostics steps for [Android](https://firebase.google.com/support/troubleshooter/fcm/delivery/diagnose/android) and [iOS](https://firebase.google.com/support/troubleshooter/fcm/delivery/diagnose/ios) in the Firebase cloud messaging documentation.
```

Comment:
Was it intended to use this ˀ sign?

Reply:
nope.. this is some grammarly autofix bug 👎